terraform/README.md

```diff
@@ -35,12 +39,12 @@ resource "juju_integration" "smf-db" {
 
```

Comment:
Can't create a suggestion, but in the line above we should replace `var.model_name` with `juju_model.my_model.name`. 
Using the occasion, the formatting of the smf module above could be improved too ;)

Reply:
Fixed